### PR TITLE
Update image/imagestream pages to use getPreferredVersion

### PR DESCRIPTION
**Pull request by benjaminapetersen**
minimal view changes, only 1/3.

Depends on PR #2173 for the first batch of `canIDoAny` updates

@spadgett

### CODE REVIEW

File: app/views/browse/imagestream.html [hunk 1]
```diff
@@ -15,10 +15,10 @@ <h1 class="contains-actions">
                class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
                data-toggle="dropdown"><i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span></a>
             <ul class="dropdown-menu dropdown-menu-right actions action-button">
-              <li ng-if="'imagestreams' | canI : 'update'">
+              <li ng-if="imageStreamsVersion | canI : 'update'">
```
Comment (spadgett) [inline app/views/browse/imagestream.html]: Needs a `canIDoAny` update

File: app/scripts/controllers/imagestream.js [hunk 2]
```diff
@@ -7,7 +7,15 @@
...
-  .controller('ImageStreamController', function ($scope, $routeParams, DataService, ProjectsService, $filter, ImageStreamsService, Navigate) {
+  .controller('ImageStreamController', function (
+    $scope,
+    $routeParams,
+    APIService,
+    DataService,
+    ProjectsService,
+    $filter,
+    ImageStreamsService,
+    Navigate) {
```
Comment (spadgett) [inline app/scripts/controllers/imagestream.js]: nit: alpha

File: app/scripts/controllers/images.js [hunk 3]
```diff
@@ -8,7 +8,15 @@
...
-  .controller('ImagesController', function ($routeParams, $scope, DataService, ProjectsService, $filter, LabelFilter, Logger) {
+  .controller('ImagesController', function (
+    $routeParams,
+    $scope,
+    APIService,
+    DataService,
+    ProjectsService,
+    $filter,
+    LabelFilter,
+    Logger) {
```
[the remark this answers is not in the document]
Comment (benjaminapetersen) [reply app/scripts/controllers/images.js]: will do.

**Comment by benjaminapetersen** [comment]
Updated, will rebase when #2173 merges.

**Comment by benjaminapetersen** [comment]
Rebased

**Comment by spadgett** [comment]
[merge]

**Pull request merged**